Symfony/CS/Config/Config.php

```diff
@@ -53,6 +54,13 @@ public function setDir($dir)
         $this->dir = $dir;
     }
 
+    public function setCacheDir($cacheDir)
```

Comment:
@GrahamCampbell @keradus 

Here you go: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1075